sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -442,9 +442,14 @@ class Scheduler {
   QueueImplPtr getDefaultHostQueue() { return DefaultHostQueue; }
 
   static MemObjRecord *getMemObjRecord(const Requirement *const Req);
+  // Virtual for testing purposes only
+  virtual void
```

Comment:
virtual methods to be removed or unit test define to be introduced to eliminate vtbl in prod